WordPress/src/main/res/layout/theme_activation_bottom_sheet.xml

```diff
@@ -0,0 +1,169 @@
+            android:layout_height="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/theme_activation_bottom_sheet_selected_description"
+            android:src="@drawable/ic_tick_vector_blue"
+            android:visibility="invisible" />
+
+        <TextView
+            android:id="@+id/use_theme_layout_sub_text_tv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/use_theme_layout_main_text_tv"
+            android:text="@string/theme_activation_use_theme_layout_sub_text"
+            android:textSize="@dimen/theme_activation_bottom_sheet_text_size_small" />
+
+        <TextView
+            android:id="@+id/use_theme_layout_main_text_tv"
+            style="@style/ThemeActionSheetOptionText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/theme_activation_use_theme_layout_main_text" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_divider_height"
+            android:layout_below="@+id/use_theme_layout_sub_text_tv"
+            android:layout_marginTop="@dimen/margin_medium"
+            android:background="@color/gray_5" />
+    </RelativeLayout>
+
+    <RelativeLayout
+        android:id="@+id/keep_current_layout_ll"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/use_theme_layout_ll"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_medium">
+
+        <ImageView
+            android:id="@+id/keep_current_check_iv"
+            android:layout_width="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/theme_activation_bottom_sheet_selected_description"
+            android:src="@drawable/ic_tick_vector_blue" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            style="@style/ThemeActionSheetOptionText"
+            android:layout_height="@dimen/theme_action_sheet_option_height"
+            android:layout_alignParentStart="true"
+            android:layout_toStartOf="@id/keep_current_check_iv"
+            android:gravity="center_vertical"
+            android:text="@string/theme_activation_keep_current_layout_main_text" />
+    </RelativeLayout>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="70dp"
```

Comment:
I think this would look better if we extract the value [as above](https://github.com/wordpress-mobile/WordPress-Android/pull/15029#discussion_r670497655).